examples/test_can/test_can.py

```diff
@@ -0,0 +1,108 @@
+##########################################################################
+# Copyright (c) 2010-2023 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+Can Auxiliary example
+*********************
+
+:module: test_can
+
+:synopsis: Example test show how send and receive can messages from can
+
+.. currentmodule:: test_can
+"""
+import logging
+import threading
+import time
+
+import pykiso
+from pykiso.auxiliaries import can_aux1, can_aux2
+from pykiso.can_message import CanMessage
+from pykiso.lib.auxiliaries.can_auxiliary import CanAuxiliary
+
+can_aux1: CanAuxiliary
+can_aux2: CanAuxiliary
+
+
+@pykiso.define_test_parameters(suite_id=1, case_id=1, aux_list=[can_aux1, can_aux2])
+class CanAuxTest(pykiso.BasicTest):
+    def setUp(self):
+        pass
+
+    def test_run(self):
```

Comment:
Normally tests have an assert statement which you don't use here.. 
I would suggest to reduce the example to only one  extra thread.
Just start tx thread and wait for incoming message in this test method. 

Do an assert on the received message where you know the signal content

Reply:
Ok, i agree, i will fix it